test/functional/test_framework/util.py

```diff
@@ -548,3 +536,13 @@ def find_vout_for_address(node, txid, addr):
         if addr == tx["vout"][i]["scriptPubKey"]["address"]:
             return i
     raise RuntimeError("Vout not found for address: txid=%s, addr=%s" % (txid, addr))
+
+
+def sendtoaddress_and_get_outpoint(node, address, amount):
```

Comment:
SGTM to make it a more generic helper that supports multiple outputs / returns multiple UTXOs (need to use `send` instead of `sendtoaddress`, but that shouldn't matter). With that, even more instances of `find_vout_address` can be replaced. Moving the helper to TestNode wouldn't be a good fit, as the outpoints creations not only operate on TestNode objects, but in some instances also on `RPCOverloadWrapper`s (result from `get_wallet_rpc`). Moving it to TestFramework should work though.